coinpaprika/README.md

```diff
@@ -131,3 +131,107 @@ Returns the global market capitilization from the [global endpoint](https://api.
   "statusCode": 200
 }
 ```
+
+## Multi Endpoint
```

Comment:
I don't love the concept of multi endpoints. I think this is adding complexity to the underlying EAs which we should avoid.

If we need two point of data from the coinpaprika `price` endpoint we should be able to just fetch a raw data payload.

**Or a solution very in-line with changes we are making** is to use a `path` parameter on the `price` endpoint for getting `price` and then `marketcap`. The cache should serve both immediately so there is low latency. One issue here is we build the cache key from the params so it would unnecessarily make two participants. Instead we could just filter out `path` from the params as a quick fix. It would see both requests hitting the `price` endpoint.

Reply:
In the case of cryptocompare, added the `multi` basically to avoid any endpoint changes on the current adapters, as the current `price` endpoint doesn't support batching. 
For the paprika case, I'll review it again, it seems that isn't necessary